### ISSUE TITLE
Explore adding an FAQ section

### ISSUE THREAD

**Issue by pradyunsg** [toml-lang/toml.io#70]
This has come up in multiple discussions over on toml-lang/toml, as a suggestion; for placing things that do not belong in the specification itself.

**Comment by pradyunsg** [comment]
A few quick thoughts worth nothing down here:

I think this:

- should NOT contain broad design guidance (eg: when to use booleans vs special "string" values).
- should only provide guidance in the specific context of TOML (eg: what to use instead of `None`).
- may also be a good place to provide information about why certain design choices were made (eg: no pragma)

The least effort way to do things would likely be to have minimal text in each answer and link to (or quote from) already provided responses on the TOML issue tracker.

**Comment by DominoPivot** [comment]
> should NOT contain broad design guidance
Yeah, It's probably for the best to keep the scope small and focus on clarifying what TOML is and isn't. Opinions that aren't actually enforced by the language or part of its design principles should probably be left out.

The obivous Qs that come to mind:
- Can I use TOML as a drop-in replacement for JSON or YAML?
- Why is there no null in TOML?
- What can I use to represent absence of a value instead of null?
